packages/backend-api/src/processing/tasks/utils.ts

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
please don't name files utils
it means nothing
and you already have a file 'util' as well :-(